docs/community/tools/strands-deepgram.md

```diff
@@ -0,0 +1,53 @@
+# strands-deepgram
```

Comment:
Could we have front-matter content on each of these packages?

Maybe:

```
project:
    pypi: https://pypi.org/project/strands-deepgram/
    github: https://github.com/eraykeskinmac/strands-deepgram 
    maintainer: eraykeskinmac
service:
    name: deepgram
    link: https://console.deepgram.com/
```

I know none of the other pages have this content, but I think going forward it would be beneficial